docs/how-to/setup-containerd-pull.md

```diff
@@ -0,0 +1,96 @@
+# Manual Steps for Setting Up Containerd and Docker for Genpolicy Tool
+
+This guide provides instructions for manually setting up your system to use the genpolicy tool with containerd and Docker. This setup is needed for managed identity-based authentication to private registries using an identity token.
+
+These are the steps performed by the script `src/tools/genpolicy/setup_containerd_docker.sh`
+
+## Prerequisites
+
+- Ensure you have `sudo` access on the system.
+- Determine the package manager used by your system (`apt-get` or `tdnf`).
+
+## Steps
+
+### 1. Install Containerd
+
+#### Using `apt-get` (for Debian/Ubuntu-based systems)
+
+```bash
+sudo apt-get update
+sudo apt-get install -y containerd
+```
+
+#### Using `tdnf` (for Photon OS-based systems)
```

Comment:
This is not an automated script, so it's OK to keep just the Ubuntu example. That's one advantage of providing a doc instead a script.